awsiot/iotshadow.py

```diff
@@ -539,8 +521,6 @@ def subscribe_to_shadow_updated_events(self, request, qos, callback):
     def subscribe_to_update_named_shadow_accepted(self, request, qos, callback):
         # type: (UpdateNamedShadowSubscriptionRequest, int, typing.Callable[[UpdateShadowResponse], None]) -> typing.Tuple[concurrent.futures.Future, str]
         """
-        API Docs: https://docs.aws.amazon.com/iot/latest/developerguide/device-shadow-mqtt.html#update-accepted-pub-sub-topic
```

Comment:
just curious, why remove this link?

Reply:
didn't touch those .peb files; will check to see if the model changed

Reply:
Follow up, looks like the doc links never got merged into mainline. I'll merge it and regenerate when I add doc strings here shortly.